app/views/pipeline_outputs/show.html.erb

```diff
@@ -38,35 +38,30 @@
 </p>
 
 <table class="table table-condensed">
-      <thead>
-        <tr>
-          <th>Taxon ID</th>
-          <th>Taxon Name</th>
-          <th>Taxon Level</th>
-          <th>Count Type</th>
-          <th>Count</th>
-        </tr>
-      </thead>
-      <tbody>
-        <% @pipeline_output.taxon_counts.sort_by(&:count).reverse.each do |element| %>
-          <tr>
-            <td><%= element.tax_id %></td>
-            <td><%= element.name %></td>
-            <td><%= case element.tax_level
-                   when TaxonCount::TAX_LEVEL_SPECIES; 'species'
-                   when TaxonCount::TAX_LEVEL_GENUS; 'genus'
-                   end %>
-            </td>
-            <td><% case element.count_type
-                   when TaxonCount::COUNT_TYPE_NT; 'nucleotide hit'
-                   when TaxonCount::COUNT_TYPE_NR; 'protein hit'
-                   end %>
-            </td>
-            <td><%= element.count %></td>
-         </tr>
-        <% end %>
-      </tbody>
-    </table>
+  <thead>
+    <tr>
+      <th>Species</th>
+      <th>NT Species Count</th>
+      <th>NR Species Count</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% counts_display = @pipeline_output.taxon_counts %>
```

Comment:
Sorry I didn't say this before, but it didn't occur to me until after posting the previous review–

This logic that deals with finding the right data should really be in either the model or controller (or a combo of both).

Templates should be as free of logic as possible, except for logic related to display/rendering/etc. 

Does that make sense? It might just mean cut/pasting this into the controller.